library/src/main/java/com/woxthebox/draglistview/BoardView.java

```diff
@@ -41,6 +41,8 @@
     public interface BoardListener {
         void onItemDragStarted(int column, int row);
 
+        void onDragItemChangedPosition(int oldColumn, int oldRow, int newColumn, int newRow);
+
```

Comment:
change name to onItemChangedPosition to follow the same style as the other methods